test/BUILD.bazel

```diff
@@ -121,20 +121,46 @@ ts_library(
     tsconfig = ":test_bundling_tsconfig.json",
     deps = [
         "//test/proto:naming_styles_ts_proto",
-        "//test/proto/common:pizza_ts_proto",
         "//test/proto:pizza_service_ts_proto",
         "//test/proto/common:delivery_person_ts_proto",
-        "@npm//google-protobuf",
+        "//test/proto/common:pizza_ts_proto",
         "@npm//@improbable-eng/grpc-web",
+        "@npm//google-protobuf",
     ],
 )
 
 jasmine_node_test(
     name = "rollup_test",
     srcs = [
-        ":rollup_test.spec.js"
+        ":rollup_test.spec.js",
     ],
     data = [
-        ":test_es6_bundling"
-    ]
+        ":test_es6_bundling",
+    ],
+)
+
+ts_library(
```

Comment:
I was a little confused why you duplicated the rollup test until I saw your comment: "because it runs on Node.js instead of the browser".

Can you change the names of both tests to indicate what's being tested? E.g. `rollup_in_node_test` and `rollup_in_browser_test`.